src/styles/selectionbar.styl

```diff
@@ -0,0 +1,65 @@
+@require 'cozy-ui'
+
+slate-grey = #5d6165
+
+.fil-selectionbar
+  position fixed
+  display flex
+  justify-content center
+  align-items center
+  width 100%
+  top 0
+  left 0
+  color white
+  background-color slate-grey
+  height em(56px)
+  font-weight bold
+  z-index 1000 // because... you know...
```

Comment:
Duuuude! 1000! really? I get that the bar should be on top of everything but… I seriously doubt that there will 1000 layers of z-index. z-index 50 should be enough, by far.